test/images/scripts/debian-testing.setup

```diff
@@ -48,14 +47,15 @@ virtinst \
 useradd -m -U -c Administrator -G sudo -s /bin/bash admin
 echo admin:foobar | chpasswd
 
-# For storaged
+# For latest udisks
 echo "deb http://deb.debian.org/debian experimental main" >/etc/apt/sources.list.d/experimental.list
 # FIXME: we need to enable jessie-backports until docker.io gets back into testing
 echo "deb http://deb.debian.org/debian jessie-backports main" >/etc/apt/sources.list.d/jessie-backports.list
 apt-get -y update
 
 export DEBIAN_FRONTEND=noninteractive
 eatmydata apt-get -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES
+eatmydata apt-get -y install -t experimental udisks2
```

Comment:
Because storaged was *only* in experimental, so apt  didn't have any other candidate for it. udisks2 is also in testing (version 2.1.8), so by default it installs that one (testing/unstable have a higher "pin value" than experimental, i. e. you have to explicitly choose packages from experimental that you want).

Note that the image, or our tests are still broken, so don't bother reviewing this yet. I just did some initial bits on the side, but I'm working on #6252 today (this PR can wait until next week).